app/states/unified_database.py

```
@@ -0,0 +1,1208 @@
+                                   end_date: Optional[date] = None, user: Optional[str] = None,
+                                   reason: Optional[str] = None) -> bool:
+        """
+        Remove a person from a committee role (set as inactive).
+        """
+        with self.get_session() as session:
+            committee_person = session.exec(
+                select(UnifiedCommitteePerson).where(
+                    UnifiedCommitteePerson.person_id == person_id,
+                    UnifiedCommitteePerson.committee_id == committee_id,
+                    UnifiedCommitteePerson.role == role,
+                    UnifiedCommitteePerson.is_active == True
```

Ruff (0.14.7)

Avoid equality comparisons to `True`; use `UnifiedCommitteePerson.is_active:` for truth checks

Replace with `UnifiedCommitteePerson.is_active`

(E712)